README.rst

```diff
@@ -34,17 +34,17 @@ $ sudo yum localinstall epel-release-7-5.noarch.rpm # Note the version may chang
  - bzip2
  - git
  - GitPython
+ - lzop
  - mock
+ - nosync
+ - python-lxml
  - PyYAML
- - svn
+ - subversion
  - wget
- - python-lxml
- - nosync
- - lzop
```

Comment:
I vote for dropping this list. Having two places to update with packages is already too much. What about we keep a file with the dependencies list and suggest a `yum install $(cat dependencies)` instead? Then your code could read from the same file to check if the packages are installed.